.ado/jobs/universal.yml

```diff
@@ -63,6 +63,11 @@
             - job: UniversalBuild${{ matrix.Name }}
               variables:
                 - template: ../variables/vs2019.yml
+
+                # Some tasks run on a different user (VssAdministrator) instead of the default user (AzDevOps).
+                # Keep NuGet cache independent from the user directory.
```

Comment:
Since we now run both jobs on the same agent spec (rnw-pool-4 and rnw-pool-8)  and no longer on the windev agents, this might no longer be needed to match between the build and the test machines.

Reply:
Let's maybe keep it in case we eventually switch to new pools that happen to use different user profiles?